CNV/cnv-upgrade.sh

```diff
@@ -2,17 +2,22 @@
 
 set -ex
 
-OLD_CNV_VERSION="${CNV_VERSION:-2.0.0}"
+OLD_CNV_VERSION="${OLD_CNV_VERSION:-2.0.0}"
 CNV_VERSION="${CNV_VERSION:-2.1.0}"
+OLD_CNV_CHANNEL="${OLD_CNV_VERSION:0:3}"
```

Comment:
can you do the same thing the above scripts?

Reply:
done